### ISSUE TITLE
Respect dark mode

### ISSUE THREAD

**Issue by sdasda7777** [ahmoylaw/RegexTrainer-Descriptions#19]
I also couldn't help but notice that your extension doesn't respect the light/dark mode settings of Notepad++. Would be cool if you improved that, it really sticks out like a sore thumb:

![image](https://github.com/ahmoylaw/RegexTrainer-Descriptions/assets/17746796/48a07f29-cadc-4176-92ad-0d6714b37150)

**Comment by CennoxX** [comment]
I could get over it in the user interface, but the add-on is becoming unusable for me in the text. The matched text should at least made black every time.
![grafik](https://github.com/ahmoylaw/RegexTrainer-Descriptions/assets/6975881/7e3ccc95-1763-4b2e-a7fa-f2d41655b689)

**Comment by BdR76** [comment]
I agree with @CennoxX  imho the gui doesn't really matter that much if it supports darkmode. 

However, the colors should really be configurable I think. As far as I can tell you currently cannot change the colors for the plugin, it has exaclty 6 colors and they seem to be hardcoded.

By default Notepad++ lexer plug-ins can store the color preference in an xml file at `%APPDATA%\Notepad++\plugins\Config\<plugin name>.xml` See how my CSV Lint plug-in creates an XML file with the color settings if the file doesn't exist, see the [TryCreateLexerXml](https://github.com/BdR76/CSVLint/blob/master/CSVLintNppPlugin/Main.cs#L236) function.

This would allow users to manually set the dark mode colors. Then the Regex Trainer could look like this:

![mission_impossible](https://github.com/user-attachments/assets/b04d55b6-16c4-413c-aa69-73f0b567a7ab)

Also, maybe it should allow more colors than just 6, like maybe 8 or 12 🤔
And you can select colors in a way that can also work for people who are color blind, see [on this page](https://github.com/BdR76/CSVLint/tree/master/extra#generate_colorspy)
